src/components/exchange/DepositInfo.tsx

```diff
@@ -100,32 +98,43 @@ export default function DepositInfo({
   });
 
   return (
-    <Animated.View>
-      <Animated.View style={animatedContainerStyle} testID="deposit-info">
+    <Box as={Animated.View}>
+      <Box
+        as={Animated.View}
+        style={animatedContainerStyle}
+        testID="deposit-info"
+      >
         <ButtonPressAnimation onPress={onPress} scaleTo={0.96}>
-          <Container>
-            <CoinIcon
-              address={asset?.address}
-              size={20}
-              symbol={asset?.symbol}
-              testID="deposit-info-container"
-            />
-            <Centered>
-              <Text
-                color={colors.alpha(darkModeThemeColors.blueGreyDark, 0.6)}
-                size="smedium"
-                weight="semibold"
-              >
+          <Box
```

Comment:
Layout components are preferred over `Box`, which may also force you to split up into separate components for each layout concern, e.g. `<Inset>` for padding and `<Inline>` to lay things out horizontally. The current docs are deployed here if you want to have a look: https://rds-jxom.vercel.app